other/babel-plugin-l10n-loader.js

```diff
@@ -0,0 +1,140 @@
+const p = require('path')
```

Comment:
It may not be a bad idea for us to add some tests around this. It's not the most complicated plugin I've written, but it's up there!